guides/common/modules/con_overview-of-recommended-disaster-recovery-plans.adoc

```diff
@@ -0,0 +1,80 @@
+[id="overview-of-recommended-disaster-recovery-plans"]
+= Overview of recommended disaster recovery plans
+
+Choose a disaster recovery plan that best helps ensure the continuity of {Project} services in your deployment.
+
+Virtualizating your {ProjectServer}::
+How do I back up?:::
+Virtualize your {ProjectServer} and use the hypervisor tools to take virtual machine snapshots of the server.
+This method is suitable if you can run {Project} on top of virtualization.
+How will I recover in case of a disruptive event?:::
+To recover {Project} services, restore a virtual machine snapshot.
++
+[IMPORTANT]
+====
+During recovery, you will need to ensure that the hostname of your {ProjectServer} does not change.
+The IP address can change.
+====
+What is the expected impact?:::
+Expect some amount of data inconsistency after recovery, based on how often you created snapshosts.
```

Comment:
What does that mean? As a user, I don't want any inconsistency! ;)

But more importantly, I need to know which kind of inconsistency we're talking about.
- Host present in Candlepin but not in Foreman?
- Host totally consistent in Foreman (and all attached services), but the knowledge is from last Sunday 12:00, when the snapshot was taken. Inconsistency with the "real world" where the host was updated/deleted on Monday?

I suspect you mean the latter, but it's not obvious to the reader.

Reply:
Good point, adding some more specific details on what kind of inconsistency users can expect here will be helpful. And while it should be obvious that it's any that's changed since the snapshot will be lost, I have found it helps users to have a level of detail below that to help them think more about impacts. Users will be trying to calculate their tolerance for data loss and use that to compare to how long a snapshot takes, how much space it takes up and thus ultimately how often they do a snapshot (or sync data in the case of the mounting of data directories).